src/components/schedule-page/table/schedule/schedule-parts/shift-cell/shift-autocomplete.component.tsx

```diff
@@ -1,34 +1,15 @@
 import React from "react";
-import { ShiftCode, shifts } from "../../../../../../common-models/shift-info.model";
+import { shifts } from "../../../../../../common-models/shift-info.model";
 import { AutocompleteComponent } from "../../../../../common-components";
 import { BaseCellInputOptions } from "../base-cell/base-cell-input.component";
 
-const shiftCodes: ShiftCode[] = [
-  ShiftCode.R,
-  ShiftCode.P,
-  ShiftCode.D,
-  ShiftCode.N,
-  ShiftCode.DN,
-  ShiftCode.PN,
-  ShiftCode.W,
-  ShiftCode.U,
-  ShiftCode.L4,
-];
-const ShiftCodeSelectItems = shiftCodes.map((shiftCode) => createSelectItem(shiftCode));
-
-function createSelectItem(shiftCode: ShiftCode): { name: string; symbol: string; code: ShiftCode } {
+const ShiftCodeSelectItems = Object.values(shifts).map((shift) => {
   return {
-    name:
-      `${shifts[shiftCode].name}` +
-      `${
-        shifts[shiftCode].isWorkingShift
-          ? `: ${shifts[shiftCode].from}-${shifts[shiftCode].to}`
-          : ""
-      }`,
-    symbol: `${shifts[shiftCode].code}`,
-    code: shiftCode,
+    name: `${shift.name} ${shift.isWorkingShift ? `: ${shift.from}-${shift.to}` : ""}`,
+    symbol: shift.code,
+    code: shift.code,
```

Comment:
Tylko jako ciekawostka, bo niedawno temu się o tym dowiedziałem 
Strzałkowe funkcje które zwracają obiekt, też można zapisać w taki sposób:

```
(shift) => ({
    name: ...
   symbol: ..
   code: ...
})
```
Ale moim zdaniem nie jest to czytelne i o wiele lepiej jest z returnem.